src/teleop/gui/src/components/GenerateReport.vue

```diff
@@ -0,0 +1,57 @@
+
+<template>
+    <div>
+        <button v-on:click="download_csv_file(spectral_data)">Generate Report</button>
```

Comment:
do we still need this button

Reply:
wait actually it might be fine since i guess ish uses it. i guess maybe go ask science to double check

Reply:
We are probably going to retool this button later on, but science still wants it.